cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/source/FileBatchSource.java

```diff
@@ -171,30 +174,30 @@ public void onRunFinish(boolean succeeded, BatchSourceContext context) {
    */
   public static class FileBatchConfig extends PluginConfig {
 
-    @Name("fileSystem")
+    @Name("Filesystem")
```

Comment:
I think it is ok to keep these as camel case. To change how they appear in the UI, add a templates file in ``cdap-ui/templates/ETLBatch`` and specify the ``Label`` field appropriately. Sync up with @ajainarayanan or @elfenheart on how to add/use those files.